### PR TITLE
Giraffe thanks page

### PR DESCRIPTION
**Pull request by jparkrr**
Depends on https://github.com/MoveOnOrg/mop-frontend/pull/330

Restyles the thanks page for giraffe
http://localhost:8080/thanks.html?petition_id=95983

tracking issue #309

### CODE REVIEW

File: src/components/theme-giraffe/raw-link.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { withShareLink } from '../../containers/hoc-share-link'
+import LinkSvg from 'GiraffeUI/svgs/link.svg'
+
+const RawLink = ({ rawLink }) => (
+  <a
+    className='mo-btn share-modal__link'
+    onClick={() => prompt('Here is your link to share:', rawLink)}
```
Comment (jparkrr) [inline src/components/theme-giraffe/raw-link.js]: Probably don't want to do this? Should we request a UX design? Should I just copy it to the clipboard?

**Comment by jparkrr** [comment]
Now ready for review!

**Comment by jparkrr** [comment]
Styling now depends on https://github.com/MoveOnOrg/giraffe/pull/78

File: src/components/theme-giraffe/raw-link.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { withShareLink } from '../../containers/hoc-share-link'
+import LinkSvg from 'GiraffeUI/svgs/link.svg'
+
+const RawLink = ({ rawLink }) => (
+  <a
+    className='mo-btn petition-thanks__link'
+    onClick={() => prompt('Here is your link to share:', rawLink)}
```
Comment (jparkrr) [inline src/components/theme-giraffe/raw-link.js]: Probably don't want to do this. Should we request a UX design? Should I just copy it to the clipboard?

Comment (schuyler1d) [reply src/components/theme-giraffe/raw-link.js]: probably. @hi0ctane?

Comment (jparkrr) [reply src/components/theme-giraffe/raw-link.js]: I filed an issue https://github.com/MoveOnOrg/giraffe/issues/79

**Comment by sreynen** [comment]
Marking this as blocked on the UX question.

**Comment by jparkrr** [comment]
@sreynen `petitionShortCode() returns a full url if you have `BASE_URL` set. You're right that it's confusing in dev though and it's probably a confusing function name.

https://github.com/MoveOnOrg/mop-frontend/blob/main/src/lib/index.js#L92-L93

**Comment by jparkrr** [comment]
Added the copy and paste stuff for the thanks page, requires https://github.com/MoveOnOrg/giraffe/pull/111

**Pull request merged**